WireUI/Sources/WireSidebarUI/Views/SidebarMenuItemView.swift

```diff
@@ -87,6 +90,11 @@ struct SidebarMenuItemView<TitleView: View>: View {
             .cornerRadius(backgroundCornerRadius)
         }
         .dynamicTypeSize(...DynamicTypeSize.accessibility1)
+        .onGeometryChange(for: CGRect.self) { proxy in
```

Comment:
I don't know what exactly happens here, but could it be that this causes performance overhead when scrolling?
I don't think every menu time view should keep track of its frame on the screen.
Maybe we can brainstorm for a bitter idea?

Reply:
Updated so that we only track the frame for the `Folders`  menu item